<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WatchlistWindow</class>
 <widget class="QMainWindow" name="WatchlistWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="addSymbolButton">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../DataAlpha.qrc">
        <normaloff>:/icons/resources/icons/buttons/add.png</normaloff>:/icons/resources/icons/buttons/add.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>256</width>
         <height>477</height>
        </rect>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2" native="true">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>170</y>
          <width>120</width>
          <height>80</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>276</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../DataAlpha.qrc"/>
 </resources>
 <connections/>
</ui>
